scripts/collect_api_events.rb

```diff
@@ -22,8 +22,8 @@ def transform(body, pe_console, source_type, splunk_client, splunk_token)
       'sourcetype' => source_type,
       'event' => pe_event,
     }
-    events = events + ' ' unless events.empty?
-    events = events + event.to_json
+    events += ' ' unless events.empty?
```

Comment:
*nit alert* If we're going to optimize whilst also stylize, I personally prefer the `<<` operator, as it appends to the original object vs creating a new one like `+=`.

Reply:
The only reason I used this method is because Robocop threw an error and specifically asked me to use it. I tested though and `<<` also resolved the error so I'm switching it now.